docs/proposals/08-peering.md

```diff
@@ -0,0 +1,168 @@
+---
+title: VNet Peering
+
+oep-number: 8
+
+creation-date: 2023-02-24
+
+status: implementable
+
+authors:
+
+- "@ManuStoessel"
+- "@gehoern"
+
+reviewers:
+
+- "@afritzler"
+- "@adracus"
+
+---
+
+# OEP-NNNN: Your short, descriptive title
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+
+Network peering is a crucial feature to allow users to route traffic between two onmetal networks. This OEP introduces a new API that allows the owner of both networks to express the consent to peering the networks explicitely and extends the existing `Network` API to reflect an active peering.
+
+- peering connects whole networks together 
+(vf get both network route set announced)
+- clear separation from transit gateways
```

Comment:
What is the separation from transit gateway and in which factors do we differentiate? What are the characteristics we expect that are different?